### ISSUE TITLE
How to replicate the BraTS 2021 results on your paper

### ISSUE THREAD

**Issue by swwzn714** [Project-MONAI/research-contributions#183]
On this github repo the results of five-fold cross validation is  

![image](https://user-images.githubusercontent.com/41735931/214499909-b3d2c780-f916-4eef-ab59-6186eebae34d.png)

which I have downloaded the model and validated the performances are consistent with the Mean Dice claimed in the table.

While reading your arxiv paper, I saw another table 

![image](https://user-images.githubusercontent.com/41735931/214500410-8e637ad6-1615-4d76-87d8-ef9e8e184727.png)

But I could not find where I can download the corresponding pretrained models.  Could you provide the pretrained models for those claims?

Best Regards

**Comment by ricklisz** [comment]
Btw have you figured this one out? Also trying to reproduce...

**Comment by Eight3H** [comment]
> On this github repo the results of five-fold cross validation is
> 
> ![image](https://user-images.githubusercontent.com/41735931/214499909-b3d2c780-f916-4eef-ab59-6186eebae34d.png)
> 
> which I have downloaded the model and validated the performances are consistent with the Mean Dice claimed in the table.
> 
> While reading your arxiv paper, I saw another table
> 
> ![image](https://user-images.githubusercontent.com/41735931/214500410-8e637ad6-1615-4d76-87d8-ef9e8e184727.png)
> 
> But I could not find where I can download the corresponding pretrained models. Could you provide the pretrained models for those claims?
> 
> Best Regards 
can  you provide  the  validation  data  json file

**Comment by RENwei-1226** [comment]
Hello , I need a json file when I use BraTS21 validation set for Evaluation, can you provide the validation data json file, and I would like to ask you, Test. py is for the segmentation of unlabeled verification data or labeled test data on the official website. Looking forward to your reply, thank you very much

**Comment by GLARKI** [comment]
I have a similar difficulty regarding reproducibility.

I've downloaded the dataset [HNSCC](https://www.cancerimagingarchive.net/collection/hnscc/).
When looking at the [json file](https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/dataset_HNSCC_0.json), I assume they are the same as the folder id, e.g., 'HNSCC-01-0001'.
However, the IDs in this JSON file go until 1200+, but in my downloaded cases, the IDs go to 630.

**Is this because the database was updated on request of the PI?**
_See quote on the database website "Version 4: Updated 2024/05/15
Replaced Head-Neck-CT-Atlas clinical data file per PI request. The old version is no longer available."._

Or did I miss something?

Thank you very much in advance!
